### PR TITLE
[INLONG-7280][Manager] rename checkXXX to expectXXX in Preconditions

### LINKED ISSUE #7280: [Improve][Manager] rename checkXXX to expectXXX in Preconditions
### Description

From: https://github.com/apache/inlong/pull/7279

image

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gosonzhang**
The reason for the change has been explained in the issue[1]: checkxxx is just an action, not an expectation; changing the name to expectXXX will clarify what results are required

1. - Fixes #7280

### CODE REVIEW

**Review by vernedeng — APPROVED** [review]
lgtm

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/DataNodeImpl.java [hunk 1]
```diff
@@ -38,36 +38,36 @@ public DataNodeImpl(ClientConfiguration configuration) {
 
     @Override
     public Integer save(DataNodeRequest request) {
-        Preconditions.checkNotNull(request, "request cannot be null");
-        Preconditions.checkNotEmpty(request.getName(), "data node name cannot be empty");
-        Preconditions.checkNotEmpty(request.getType(), "data node type cannot be empty");
+        Preconditions.expectNotNull(request, "request cannot be null");
+        Preconditions.expectNotEmpty(request.getName(), "data node name cannot be empty");
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/DataNodeImpl.java:42]: `Preconditions.expectNotBlank`.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/DataNodeImpl.java [hunk 2]
```diff
@@ -38,36 +38,36 @@ public DataNodeImpl(ClientConfiguration configuration) {
...
     }
 
     @Override
     public Boolean update(DataNodeRequest request) {
-        Preconditions.checkNotNull(request, "request cannot be null");
-        Preconditions.checkNotEmpty(request.getName(), "data node name cannot be empty");
-        Preconditions.checkNotEmpty(request.getType(), "data node type cannot be empty");
-        Preconditions.checkNotNull(request.getId(), "data node id cannot be null");
+        Preconditions.expectNotNull(request, "request cannot be null");
+        Preconditions.expectNotEmpty(request.getName(), "data node name cannot be empty");
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/DataNodeImpl.java:62]: `Preconditions.expectNotBlank`.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/HeartbeatImpl.java [hunk 3]
```diff
@@ -45,55 +45,55 @@ public HeartbeatImpl(ClientConfiguration configuration) {
...
     }
 
     @Override
     public GroupHeartbeatResponse getGroupHeartbeat(HeartbeatQueryRequest request) {
-        Preconditions.checkNotNull(request, ErrorCodeEnum.REQUEST_IS_EMPTY.getMessage());
-        Preconditions.checkNotEmpty(request.getComponent(), ErrorCodeEnum.REQUEST_COMPONENT_EMPTY.getMessage());
-        Preconditions.checkNotEmpty(request.getInstance(), ErrorCodeEnum.REQUEST_INSTANCE_EMPTY.getMessage());
-        Preconditions.checkNotEmpty(request.getInlongGroupId(), ErrorCodeEnum.GROUP_ID_IS_EMPTY.getMessage());
+        Preconditions.expectNotNull(request, ErrorCodeEnum.REQUEST_IS_EMPTY.getMessage());
+        Preconditions.expectNotEmpty(request.getComponent(), ErrorCodeEnum.REQUEST_COMPONENT_EMPTY.getMessage());
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/HeartbeatImpl.java:58]: `Preconditions.expectNotBlank`.

File: inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/HeartbeatImpl.java [hunk 4]
```diff
@@ -45,55 +45,55 @@ public HeartbeatImpl(ClientConfiguration configuration) {
...
 
     @Override
     public StreamHeartbeatResponse getStreamHeartbeat(HeartbeatQueryRequest request) {
-        Preconditions.checkNotNull(request, ErrorCodeEnum.REQUEST_IS_EMPTY.getMessage());
-        Preconditions.checkNotEmpty(request.getComponent(), ErrorCodeEnum.REQUEST_COMPONENT_EMPTY.getMessage());
-        Preconditions.checkNotEmpty(request.getInstance(), ErrorCodeEnum.REQUEST_INSTANCE_EMPTY.getMessage());
-        Preconditions.checkNotEmpty(request.getInlongGroupId(), ErrorCodeEnum.GROUP_ID_IS_EMPTY.getMessage());
-        Preconditions.checkNotEmpty(request.getInlongStreamId(), ErrorCodeEnum.STREAM_ID_IS_EMPTY.getMessage());
+        Preconditions.expectNotNull(request, ErrorCodeEnum.REQUEST_IS_EMPTY.getMessage());
+        Preconditions.expectNotEmpty(request.getComponent(), ErrorCodeEnum.REQUEST_COMPONENT_EMPTY.getMessage());
```
Comment (fuweng11) [inline inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/HeartbeatImpl.java:68]: `Preconditions.expectNotBlank`.

**Comment by gosonzhang** [comment]
@fuweng11, for the problem of using unreasonable api for inspection in the implementation, let's modify the bill of lading separately, this bill is only for equivalent replacement

**Review by fuweng11 — APPROVED** [review]

**Comment by dockerzhang** [comment]
@gosonzhang please add more details about why we need to rename checkXXX to expectXXX.

**Comment by gosonzhang** [comment]
@dockerzhang, the reason for the change has been explained in the issue[1]: checkxxx is just an action, not an expectation; changing the name to expectXXX will clarify what results are required

1. https://github.com/apache/inlong/issues/7280

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**